test/fixtures/usersegment/index.html

```diff
@@ -39,8 +39,14 @@
       keys: {
         email: 'someone@somewhere.com'
       }
-    }, function (keys) {
-      if (keys !== undefined) {
+    }, function (response) {
+      if (response &&
+          response.length &&
+          response[0].key !== undefined &&
+          response[0].attributes !== undefined &&
+          response[0].extraFields !== undefined &&
```

Comment:
I found that `extraFields` is a bug...Personalization API now returns `extraFields: {audienceId: "999"}` but it should be `audienceId: "999"` in fact. I'll deploy a fix.

Reply:
https://github.com/treasure-data/td-cdp-kvs-server/pull/108